ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java

```diff
@@ -119,6 +119,7 @@
 
     private CCDDocument draftOrderDoc;
     private CCDDocument draftLetterDoc;
+    private CCDDocument coverLetterDoc;
```

Comment:
Rather than an individual field this should be included in the List `caseDocuments`

Reply:
@dharmendrak please respond on this one